src/config/Config.zig

```diff
@@ -1073,7 +1073,12 @@ keybind: Keybinds = .{},
 
 /// The font that will be used for the application's window and tab titles.
 ///
-/// This is currently only supported on macOS.
+/// If this setting is left unset, the default font will be used. On macOS
+/// this is "SF Pro." On Gnome it is usually "Cantarell" but it is possible to
```

Comment:
I recommend removing what the default font is. Any system could easily change this and I don't want to deal with maintaining it. Your note that it is the system default is good enough I think.